recipes/acorns/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "acorns" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deshanadesai/autodiff/archive/v0.0.1.tar.gz
+  sha256: 64c1de7798d247b3987b886b84da367d189a7debe4670c9f9d687a67c4eb7ce0
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - setuptools
```

Comment:
You probably should remove setuptools here. It is implied by pip below

Reply:
Edited. Thanks!